### ISSUE TITLE
Allowing Continued Development Through VT GitHub Organization 

### ISSUE THREAD

**Issue by mpetracca26** [jpurviance/Hardware-Cache-Simulation-and-Algorithm-Evaluation#5]
You should consider adding this to the VT GitHub Organization to increase its exposure and allow other VT students to continue to develop it.

**Comment by jpurviance** [comment]
I agree. I feel that the documentation could use some improvements first.

**Comment by mpetracca26** [comment]
Yes, I think that's a good idea.  It could use some directions on how to install and run the simulator.
